journalbeat/docs/fields.asciidoc

```diff
@@ -0,0 +1,4854 @@
+
```

Comment:
This is just a placehold (contains filebeat fields) until I can run make update.